3rdparty/glog/BUILD

```diff
@@ -8,11 +8,38 @@ package_version = "0.3.4"
 package_file = package_name + "-" + package_version + ".tar.gz"
 package_dir = package_name + "-" + package_version
 
+COMMON_EXECUTE = [
+    "export UNWIND_DIR=$$(pwd)/$(GENDIR)/3rdparty/libunwind",
+    "export INSTALL_DIR=$$(pwd)/$(@D)",
+    "export TMP_DIR=$$(mktemp -d -t glog.XXXXX)",
+    "mkdir -p $$TMP_DIR",
+    "cp -R $(SRCS) $$TMP_DIR",
+    "cd $$TMP_DIR",
+    "tar xfz " + package_file,
+    "cd " + package_dir,
+]
+
+DARWIN_EXECUTE = "\n".join(COMMON_EXECUTE + [
+    "./configure --prefix=$$INSTALL_DIR --enable-shared=no",
+    "make install",
+    "rm -rf $$TMP_DIR",
+])
+
+LINUX_EXECUTE = "\n".join(COMMON_EXECUTE + [
+     './configure --prefix=$$INSTALL_DIR --enable-shared=no LIBS="-Wl,--rpath -Wl,$$UNWIND_DIR/lib -L$$UNWIND_DIR/lib" CPPFLAGS=-I$$UNWIND_DIR/include LDFLAGS=-L$$UNWIND_DIR/lib',
+     'make install LIBS="-Wl,--rpath -Wl,$$UNWIND_DIR/lib -L$$UNWIND_DIR/lib" CPPFLAGS=-I$$UNWIND_DIR/include LDFLAGS=-L$$UNWIND_DIR/lib',
+     'rm -rf $$TMP_DIR',
+])
```

Comment:
This block above seems to be the same as that in gperftools/BUILD. Any way we can share this and not duplicate logic?